### DISCUSSION TITLE
Is there a way to set the directory for the .verified files globally

### DISCUSSION THREAD

**Discussion by mickvikt** [VerifyTests/Verify#625] [Q&A] [answered]
Hello, I managed to do this by creating a derived class for the `VerifyBase`, overloading the `Verify `method and inheriting my test class from that new class.

But is there a way to set the directory for ALL tests in the project via the `[ModuleInitializer]`? I can't find a method for setting the directory in the `VerifierSettings `class.

Thank you

**Comment by SimonCropp** [comment, marked as answer]
https://github.com/VerifyTests/Verify/blob/main/docs/naming.md#derivepathinfo
